Services/ReportsRepository/classes/class.catXLSXWriter.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+
+require_once 'Services/ReportsRepository/interfaces/interface.ExcelWriter.php';
+
+use Box\Spout\Writer\WriterFactory;
+use Box\Spout\Common\Type;
+use Box\Spout\Writer\Style\StyleBuilder;
+
+class catXLSXWriter implements ExcelWriter {
```

Comment:
Ah, habs verstanden. Wir nehmen nicht an, das der ExcelWriter ein XLSX erzeugt. Bitte dann trotzdem den catXLSXWriter in SpoutXLSXWriter umbenennen.

BTW: Was passiert eigentlich, wenn ich offerDownload("foo.XLS") auf dem SpoutXLSXWriter sage?